inc/front/cdn.php

```diff
@@ -57,6 +57,32 @@ function rocket_cdn_file( $url ) {
 	return $url;
 }
 
+add_filter( 'wp_get_attachment_image_src', 'rocket_cdn_attachment_image_src', PHP_INT_MAX );
+/**
+ * Replace URL by CDN of images displayed using wp_get_attachment_image_src
+ *
+ * @since 2.9.2
+ * @author Remy Perona
+ *
```

Comment:
Could we add a @source comment with this link as a part of the code provided from an other contributor? => https://github.com/wp-media/wp-rocket/issues/271#issuecomment-269849927